From: Jakub Jirutka <jakub@jirutka.cz>
Date: Fri, 10 Mar 2017 00:29:30 +0100
Subject: [PATCH] Fix error when calling search result iterator

This patch fixes error:

    tests/test.lua:175: bad argument #1 to 'iter' (table expected, got no value)

Upstream-Issue: https://github.com/bdellegrazie/lualdap/pull/1
--- a/src/lualdap.c
+++ b/src/lualdap.c
@@ -726,8 +726,6 @@
 	int rc;
 	int ret;
 
-	luaL_checktype(L, 1, LUA_TTABLE);
-
 	lua_rawgeti (L, LUA_REGISTRYINDEX, search->conn);
 	conn = (conn_data *)lua_touserdata (L, -1); /* get connection */
 
